__tests__/server/middleware/createRequestHtmlFragment.spec.js

```diff
@@ -272,7 +263,7 @@ describe('createRequestHtmlFragment', () => {
     console.error = jest.fn();
     middleware(req, res, next);
     expect(console.error).toHaveBeenCalled();
-    expect(console.error.mock.calls[0]).toEqual(['error creating request HTML fragment for http://example.com/request', error]);
+    expect(console.error.mock.calls[0][0]).toEqual('error creating request HTML fragment for http://example.com/request');
```

Comment:
Had to modify this test case as the same error generated above was different in node 18 than whats generated within the function.